index.js

```diff
@@ -169,25 +191,60 @@ function fastifyMultipart (fastify, options, done) {
     })
   }
 
-  const defaultThrowFileSizeLimit = typeof options.throwFileSizeLimit === 'boolean'
-    ? options.throwFileSizeLimit
-    : true
-
-  const PartsLimitError = createError('FST_PARTS_LIMIT', 'reach parts limit', 413)
-  const FilesLimitError = createError('FST_FILES_LIMIT', 'reach files limit', 413)
-  const FieldsLimitError = createError('FST_FIELDS_LIMIT', 'reach fields limit', 413)
-  const RequestFileTooLargeError = createError('FST_REQ_FILE_TOO_LARGE', 'request file too large, please check multipart config', 413)
-  const PrototypeViolationError = createError('FST_PROTO_VIOLATION', 'prototype property is not allowed as field name', 400)
-  const InvalidMultipartContentTypeError = createError('FST_INVALID_MULTIPART_CONTENT_TYPE', 'the request is not multipart', 406)
-  const InvalidJSONFieldError = createError('FST_INVALID_JSON_FIELD_ERROR', 'a request field is not a valid JSON as declared by its Content-Type', 406)
+  const defaultThrowFileSizeLimit =
+    typeof options.throwFileSizeLimit === 'boolean'
+      ? options.throwFileSizeLimit
+      : true
+
+  const PartsLimitError = createError(
+    'FST_PARTS_LIMIT',
+    'reach parts limit',
+    413
+  )
+  const FilesLimitError = createError(
+    'FST_FILES_LIMIT',
+    'reach files limit',
+    413
+  )
+  const FieldsLimitError = createError(
+    'FST_FIELDS_LIMIT',
+    'reach fields limit',
+    413
+  )
+  const RequestFileTooLargeError = createError(
+    'FST_REQ_FILE_TOO_LARGE',
+    'request file too large, please check multipart config',
+    413
+  )
+  const PrototypeViolationError = createError(
+    'FST_PROTO_VIOLATION',
+    'prototype property is not allowed as field name',
+    400
+  )
+  const InvalidMultipartContentTypeError = createError(
+    'FST_INVALID_MULTIPART_CONTENT_TYPE',
+    'the request is not multipart',
+    406
+  )
+  const InvalidJSONFieldError = createError(
+    'FST_INVALID_JSON_FIELD_ERROR',
+    'a request field is not a valid JSON as declared by its Content-Type',
+    406
+  )
+  const FileBufferNotFoundError = createError(
+    'FST_FILE_BUFFER_NOT_FOUND',
+    'the file buffer was not found',
+    500
+  )
```

Comment:
is this change relevant? Seems just linting

Reply:
Sorry, linter went crazy, I think they're all solved now